src/pages/patientView/PatientViewPage.tsx

```diff
@@ -486,7 +514,24 @@ export default class PatientViewPage extends React.Component<
 
     @action.bound
     onCnaTableRowClick(d: DiscreteCopyNumberData[]) {
-        if (d.length) {
+        // if one cna, select it and toggle off other selected
+        if (d.length == 1) {
+            this.patientViewCnaDataStore.setSelectedCna(d);
+            if (this.patientViewMutationDataStore.selectedMutations[0]) {
```

Comment:
just use arr.length

Reply:
or `isEmpty` function from `lodash`

Reply:
also, try to always use trip equals ===
this avoids a feature of javascript called type coercion.    it's just a good practice